pages.de/common/kubectl-delete.md

```diff
@@ -0,0 +1,32 @@
+# kubectl delete
+
+> Löschen von Kubernetes-Ressourcen.
```

Comment:
We always write descriptions in imperative mode.
```suggestion
> Lösche Kubernetes-Ressourcen.
```
Please also change this in the other descriptions.